package.json

```diff
@@ -33,8 +33,7 @@
     "std-version": "standard-version -m \"chore(release): version %s build ${TRAVIS_BUILD_NUMBER} [ci skip]\"",
     "test:coverage": "jest --coverage",
     "test:coveralls": "jest --coverage && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
-    "test:dev": "jest",
-    "test": "node scripts/test.js",
```

Comment:
The `test.js` script provided a nice "watch" mode for unit tests.  I'm not quite sure if it was the script or if it's something in `jest` or a combination of both.  I'm okay removing the script if we could restore that watch functionality.